jena-arq/src/main/java/org/apache/jena/riot/RDFParser.java

```diff
@@ -96,7 +96,8 @@
     private final ErrorHandler      errorHandler;
     private final Context           context;
 
-    private boolean                 canUse = true;
+    // Some cases the parser is reusable (read a file), some are not (input streams).
+    private boolean                 canUseThisPrser = true;
```

Comment:
`s/Prser/Parser`... That's an internal variable, so no biggie.

Reply:
Still worth changing. Fixed in the forced push below.